fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/CompostableItemRegistry.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.Item2ObjectMap;
+import net.fabricmc.fabric.impl.content.registry.CompostableItemRegistryImpl;
+
+/**
+ * Registry of Items to 0.0-1.0 values, defining the chance of a given item
+ * increasing the Composter block's level.
+ */
+public interface CompostableItemRegistry {
+	Item2ObjectMap<Float> INSTANCE = CompostableItemRegistryImpl.INSTANCE;
```

Comment:
Using the existing Object2FloatMap ?

Reply:
It's a pretty different thing. Map may not be the right word for the class, but the interface is unique